src/main/proto/marathon.proto

```diff
@@ -316,4 +317,19 @@ message ResidencyDefinition {
 message Secret {
   required string id = 1;
   required string source = 2;
-}
\ No newline at end of file
+}
+
+message EnvVarReference {
+  enum Type {
+    UNKNOWN = 0;
```

Comment:
pattern adopted from the core team. useful when the enum changes in a backwards incompatible way; you want a sane default value for enum values that don't exist any more. in this case, the sane value is UNKNOWN.